docs/02.API-REFERENCE.md

```diff
@@ -2126,6 +2126,65 @@ jerry_value_is_boolean (const jerry_value_t value)
 
 - [jerry_release_value](#jerry_release_value)
 
+## jerry_value_is_true
+
+```c
+bool
+jerry_value_is_true (const jerry_value_t value)
+```
+
+- `value` - api value
+- return value
```

Comment:
`return value` -> `return bool`

Reply:
Moreover, as I see thats an error in all `jerry_value_is_xxx` function descriptions, so it'd be great to fix it in a followup patch.

Reply:
Sorry, I've missed that part, `return value` is a category, not an annotation for return type. Please ignore this comment and revert the corresponding changes.

Reply:
No problem. I will revert it.